app/components/Header/Header.css

```diff
@@ -113,70 +122,84 @@
   align-items: center;
   flex: 2;
   white-space: nowrap;
-  margin-right: 5px;
+  margin-right: 0.3em;
 
   @media (--mobile-device) {
     display: none;
   }
 }
 
-.navigation a {
-  font-size: 1.35rem;
-  padding: 10px 1rem;
-  height: 100%;
-  color: var(--color-dark-gray-5);
-  text-transform: uppercase;
+.navbarEl {
+  margin: auto;
+}
+
+.navbarList {
   display: flex;
-  align-items: center;
   justify-content: center;
-  position: relative;
-  transition: color 0.2s;
-  white-space: nowrap;
+  list-style: none;
+  margin: 0;
+}
 
-  &::after {
-    content: '';
-    height: 10px;
-    left: 0;
-    right: 0;
-    border-bottom: 2px solid #c24538;
-    border-radius: 5px;
-    position: absolute;
-    bottom: 0;
-    transform: scaleX(0);
-    transition: all 0.2s ease-in-out;
-  }
+.navbarItemEl {
+  position: relative;
+  width: max-content;
+}
 
-  &:hover,
-  &.activeItem {
-    color: var(--color-black);
-  }
+.navbarItemTitle {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  height: 32.31px; /* The height of the Abakus logo */
+  padding: 0 1.5rem 0 1.5rem;
+  font-family: -apple-system, system-ui, 'Segoe UI', 'Roboto', 'Helvetica Neue',
+    Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  font-size: 1.3em;
+  color: var(--color-dark-gray-4);
+  background: transparent;
+  border: 0;
+  cursor: pointer;
+  z-index: 2;
+  transition: opacity 250ms ease-in-out;
+}
 
-  &:hover::after,
-  &.activeItem::after {
-    transform: scaleX(1);
-    bottom: 3px;
-  }
+.navbarItemTitle:hover,
+.activeNavbarItemTitle {
+  opacity: 0.7;
+}
```

Comment:
WIth all the navbars having the same hover style, I find it really weird when hovering over the one with no dropdown (karriere). Since it looks just the same, I kind of expect it to show a menu and just wait for it for a second before realizing there is none. Maybe we could do something to inform the user that it's just a plain link and won't show a dropdown? 

This isn't a priority though, just a little picky thing I noticed.

Reply:
It's a valid point, and something I've thought a lot about. I thought it would be unnecessary to add a unique dropdown to  when not really needed, as it would be more annoying than beneficial. This is also the same approach other websites that utilize a similar navbar have gone for, e.g. [stripe.com](https://stripe.com) where the "Pricing" does not have its own dropdown. I know I've seen others do the same. Now regarding giving the user a different feedback on it ... I'm not sure, as I can see how it would make the link feel more "out of place" 🤔 Thoughts?

Reply:
I'll add that stripe actually differentiates the hover effects slightly by changing the cursor, but I like the fact that the navigation links (that people are used to and love) still works as they expect them to and takes them to the same place as before, so that they are not "forced" to utilize the new dropdown. People don't usually like change, and therefore it's essential to not try to fix something that is not broken.